File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/ChargeInvoiceRows.js
goog.provide('API.Client.ChargeInvoiceRows');

/**
 * Billing invoices associated with a service, including invoice IDs, descriptions, amounts, and payment status.
 * @record
 */
API.Client.ChargeInvoiceRows = function() {}

/**
 * Whether the invoice retrieval was successful.
 * @type {!boolean}
 * @export
 */
API.Client.ChargeInvoiceRows.prototype.success;

/**
 * List of invoices for the service.
 * @type {!Object<!string, API.Client.ChargeInvoiceRows_invoices_value>}
 * @export
 */
API.Client.ChargeInvoiceRows.prototype.invoices;

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/ChargeInvoiceRows.scala
package io.swagger.server.model


/**
 * Billing invoices associated with a service, including invoice IDs, descriptions, amounts, and payment status.
 *
 * @param success Whether the invoice retrieval was successful.
 * @param invoices List of invoices for the service.
 */
case class ChargeInvoiceRows (
  success: Option[Boolean],
  invoices: Option[Map[String, ChargeInvoiceRows_invoices]]
)

